opencti-platform/opencti-front/src/private/components/trash/all/DeleteOperationLine.tsx

```diff
@@ -0,0 +1,175 @@
+import React from 'react';
+import { createFragmentContainer, graphql } from 'react-relay';
+import ListItem from '@mui/material/ListItem';
+import ListItemIcon from '@mui/material/ListItemIcon';
+import ListItemText from '@mui/material/ListItemText';
+import Checkbox from '@mui/material/Checkbox';
+import Skeleton from '@mui/material/Skeleton';
+import { MoreVert } from '@mui/icons-material';
+import makeStyles from '@mui/styles/makeStyles';
+import IconButton from '@mui/material/IconButton';
+import ListItemSecondaryAction from '@mui/material/ListItemSecondaryAction';
+import { DeleteOperationsLinesPaginationQuery$variables } from './__generated__/DeleteOperationsLinesPaginationQuery.graphql';
+import { DeleteOperationLine_node$data } from './__generated__/DeleteOperationLine_node.graphql';
+import DeleteOperationPopover from './DeleteOperationPopover';
+import { useFormatter } from '../../../../components/i18n';
+import ItemIcon from '../../../../components/ItemIcon';
+import type { Theme } from '../../../../components/Theme';
+import { DataColumns } from '../../../../components/list_lines';
+import ItemEntityType from '../../../../components/ItemEntityType';
+
+const useStyles = makeStyles((theme: Theme) => ({
+  item: {
+    paddingLeft: 10,
+    height: 50,
+  },
+  itemIcon: {
+    color: theme.palette.primary.main,
+  },
+  bodyItem: {
+    height: 20,
+    fontSize: 13,
+    float: 'left',
+    whiteSpace: 'nowrap',
+    overflow: 'hidden',
+    textOverflow: 'ellipsis',
+    paddingRight: 10,
+  },
+  itemIconDisabled: {
+    color: theme.palette.grey?.[700],
+  },
+}));
```

Comment:
We try to avoid `makeStyles` in new code as it is deprecated. You could use `style` or `sx` properties instead